desktop-exporter/app/components/detail-view/detail-view.tsx

```diff
@@ -1,9 +1,11 @@
 import React from "react";
 import { Flex, Tab, TabList, TabPanels, Tabs } from "@chakra-ui/react";
 
+
```

Comment:
nit, extra line. Prettier should consolidate these I thought?

Reply:
It treats lines between imports as intentional for some reason. Not sure why. I'll be more careful.